file-formats/smbc/examples/z_aff_example_smbc.smbc.json

```diff
@@ -0,0 +1,23 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "AFF Example Object for SMBC",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "standard"
```

Comment:
`"abapLanguageVersion": "standard"` shouldn't be generated, should it?

Reply:
you are right, we have to change the annotation for this field. See my comment below